### ISSUE TITLE
mssql / tedious

### ISSUE THREAD

**Issue by MarkTiedemann** [denoland/wanted_modules#19]
See https://github.com/tediousjs/tedious/issues/1246

**Comment by JHarrisGTI** [comment]
Yes please! I'm trying to switch my company's APIs over to using Deno and lack of MSSQL compatibility is a blocker for us.

**Comment by v1d3rm3** [comment]
Same here, the problem of Deno is the bad ecosystem.
